users/decorator.py

```diff
@@ -0,0 +1,35 @@
+import jwt
+import json
+
+from django.http import (
+    JsonResponse,
+    HttpResponse
+)
+
+from users.models import User
+from dr_martens.settings import (
+    SECRET_KEY,
+    ALGORITHM
+)
+
+def login_check(func):
+    def wrapper(self, request, *args, **kwargs):
+        access_token = request.headers.get("Authorization", None)
+        if access_token is not None:
+            try:
+                decode_token  = jwt.decode(access_token, SECRET_KEY, algorithms = ALGORITHM)
+                users_id      = decode_token['users_id']
```

Comment:
```suggestion
                user_id      = decode_token['user_id']
```
사소하지만 user_id로 하시면 더 좋을것 같습니다. 토큰 생성시 전달하는 코드에도 수정해주세요.